sdk/src/oauth/state-machines/debug-oauth-2025-03-26.ts

```
@@ -1019,26 +1063,13 @@ export const createDebugOAuthStateMachine = (
-              if (scopesSupported?.includes("offline_access")) {
-                scopes.add("offline_access");
-              }
-
-              if (scopes.size > 0) {
-                authUrl.searchParams.set("scope", Array.from(scopes).join(" "));
-              }
+            const requestedScopeValue = resolveRequestedScopeValue({
+              customScopes,
+              supportedScopes:
+                state.authorizationServerMetadata.scopes_supported,
+            });
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)